resources/views/maps/groups/_form.blade.php

```diff
@@ -16,5 +16,5 @@
 
 <div class="form-group">
     <label>{{ __('maps/groups.fields.position') }}</label>
-    {!! Form::number('position', null, ['placeholder' => __('maps/groups.placeholders.position'), 'class' => 'form-control', 'maxlength' => 3]) !!}
+    {!! Form::select('position', isset($map) ? $map->groupPositionOptions() : [], (!empty($group) ? $group : null), ['class' => 'form-control']) !!}
```

Comment:
Default when adding a new group should be the last option being selected, rather than the first.

Reply:
Also, the "name" field is missing the `required` attribute, to avoid sending the form without filling it out.